flink-ai-flow/examples/wide_and_deep/workflows/wdl/wdl.py

```diff
@@ -78,20 +84,29 @@ def stream_train():
                       model_info=stream_model_info, name='census_stream_push')
 
 
-def stream_prediction():
+def stream_prediction(use_jar_preprocess):
     """Stream Prediction Jobs"""
     stream_preprocess_input = af.get_dataset_by_name('stream_preprocess_input')
     stream_predict_input = af.get_dataset_by_name('stream_predict_input')
     stream_predict_output = af.get_dataset_by_name('stream_predict_output')
 
     stream_model_info = af.get_model_by_name('wide_and_deep')
     with af.job_config(job_name='census_stream_preprocess_predict'):
-        stream_predict_preprocess_source = af.read_dataset(dataset_info=stream_preprocess_input,
-                                                           read_dataset_processor=StreamPredictPreprocessSource())
-        stream_predict_preprocess_sink = af.user_define_operation(
-            input=[stream_predict_preprocess_source],
-            processor=StreamPredictPreprocessSink(),
-            name='census_stream_predict_preprocess_sink')
+        if use_jar_preprocess:
+            preprocess_jar_filename = "wdl-preprocess-0.1-SNAPSHOT.jar"
+            check_jar_dependencies(preprocess_jar_filename)
+            af.user_define_operation(processor=FlinkJavaProcessor(entry_class="org.aiflow.StreamPredictPreprocess",
```

Comment:
Do we have an assumption that we must use `user_define_operation()` for java processors?